backend/src/main/java/com/bakdata/conquery/models/query/queryplan/aggregators/specific/value/RandomValueAggregator.java

```diff
@@ -18,7 +19,7 @@
 @ToString(callSuper = true, onlyExplicitlyIncluded = true)
 public class RandomValueAggregator<VALUE> extends SingleColumnAggregator<VALUE> {
 
-	private final Random random = new Random();
+	private final Random random = new Random(ConqueryConstants.RANDOM_SEED);
 	private int event;
```

Comment:
Klappt das überhaupt mit konstanten ergebnissen? Wir haben doch keine Kontrolle darüber welche entities vom QueryExecutor in diesen Aggregator gesteckt werden, oder? Das ist doch immer ein Race-Condition

Reply:
hier hat sich nichts geändert, oder?